Wrong image displayed for Interrupt Vector

omkar98 opened the pull request:
Commit 2: Uploaded the image representing the Interrupt Vector with the naming conventions as given for other images.
Commit 1: Replaced the image.

Referred to the following page number in the reference book: Page: 163

The details of the book I have used for reference: 
THE DESIGN OF THE UNix® OPERATING SYSTEM 
by Maurice J. Bach
ISBN-978-81-203-Q516-8
Indian edition published by PHI Learning Private Limited. 
Published by Asoke K. Ghosh, PHI Learning Private Limited, M-97, Connaught
Circus, New Delhi-110001 and Printed by Baba Barkha Nath Printers, Bahadurgarh,
Haryana-124507.

omkar98 committed:
Placed interrupt vector image

Interrupt vector was represented with an image of the Context of a Process. Replaced it with the actual screenshot.

omkar98 committed:
Uploading the actual image of Interrupt Vector

The name of the image is given with the convention used for naming other images.

omkar98 committed:
spelling - inode

corrected spelling

suvratapte commented:
Thank you so much Omkar for your pull request. :)
There are a few things however:

Commit 1:
Can you rename it as: "Fix: Spelling of inode".

Commit 2 and 3:
You should squash commit 2 and 3 into one commit and name it as: "Fix: Interrupt vector image".

Let me know what do you think :)

omkar98 commented:
Sure Suvrat! Will do the necessary!

omkar98 committed:
Fix: Spelling of other

suvratapte commented on 1-General-Overview-of-the-System.md:
```diff
@@ -71,4 +71,4 @@ The kernel sometimes wants to prevent occurrence of interrupts when it is doing
 
 ### Memory Management
 
-The kernel permanently resides in the main memory. When compiling a program, the compiler generates a set of addresses in the program that represent addresses of variables and data structures of the addresses of instructions such as functions. The compiler generates the addresses for a *virtual machine* as if no other program will execute simultaneously on the physical machine. When the program i ...
```
I don't see any change here. Maybe only the line ending has changed. Will you please remove this change and resubmit? Thanks @omkar98 for your help! :)

omkar98 replied:
Oh! Yes. There's no change here. I did that by mistake..

suvratapte commented on 1-General-Overview-of-the-System.md:
```diff
@@ -2,7 +2,7 @@
 
 ## Why UNIX became so popular
 
-* The system is written in a high-level language, making it easy to read, understand, change, and move to tother machines.
```
Thank you so much for spotting this! :)

suvratapte commented:
Hi @omkar98, thanks for the new commit :)
Will you please update the last commits last well? If you want, we can get on a Hangouts call and make the required changes. Let me know.

omkar98 committed:
Fix: Minor Mistake

omkar98 committed:
Fix: Typo error

omkar98 commented:
> 
> 
> Hi @omkar98, thanks for the new commit :)
> Will you please update the last commits last well? If you want, we can get on a Hangouts call and make the required changes. Let me know.

omkar98 commented:
Oh! I am finding few issues with my git! I am new to GitHub, and used GitLab earlier. Anyways, will squash the previous commits as you mentioned earlier and will do the necessary changes.

omkar98 committed:
Fix: Typo error and a Correction

1. contexts with contents
2. As mentioned in the book (pg. 41): Hence, the buffers that are closer to the head of the free list have not been used as recently as those that are further from the head of the free list.

omkar98 committed:
Fix: Typo Error

has to hash.
full stop at the end.
